service/rosetta.go

```diff
@@ -8,9 +8,9 @@ import (
 
 var NodeVersion = fmt.Sprintf(
 	"%d.%d.%d",
-	version.Current.Major(),
-	version.Current.Minor(),
-	version.Current.Patch(),
+	version.Current.Major,
+	version.Current.Minor,
+	version.Current.Patch,
```

Comment:
Are we supposed to bump the `MiddlewareVersion` too?

Reply:
I usually do that once this is merged.

Reply:
but probably better for @rajranjan0608 to do it here.

Reply:
Sure thing.

Reply:
https://github.com/ava-labs/avalanche-rosetta/blob/05c01b41a3dcad4c217212e49fbeb4f25311bb91/service/rosetta.go#L17